wsgi/kuma.wsgi

```diff
@@ -29,6 +29,10 @@ import manage
 from django.core.wsgi import get_wsgi_application
 django_app = get_wsgi_application()
 
+# Enable WhiteNoise
+from kuma.core.whitenoise import KumaDjangoWhiteNoise
```

Comment:
I think this should have been `from kuma.core.whitenoise.KumaDjangoWhiteNoise import KumaDjangoWhiteNoise` for the case in which the file is named `KumaDjangoWhiteNoise.py`. But if you change the `KumaDjangoWhiteNoise` class to be contained in `kuma.core.whitenoise` then this is correct.